src/amici.cpp

```diff
@@ -1424,7 +1794,11 @@ int workForwardProblem(UserData *udata, TempData *tdata, ReturnData *rdata, cons
                         }
                     } else {
                         if (udata->nx>0) {
-                            status = AMISolve(ami_mem, RCONST(udata->ts[it]), tdata->x, tdata->dx, &(tdata->t), AMICI_NORMAL);
+                            if (isinf(udata->ts[it])) {
+                                status = workSteadyStateProblem(udata, tdata, rdata, ami_mem, it);
```

Comment:
you don't need that, simply specifying -inf as starting time should do the trick. You cannot preequilibrate sensitivities. If you preequilibrate and want to compute sensitivities in the following, you need to (always) initialize the sx0 with the steady state sensitivities.